modules/installation-creating-azure-control-plane.adoc

```diff
@@ -58,13 +58,11 @@ $ az deployment group create -g ${RESOURCE_GROUP} \
   --template-file "<installation_directory>/05_masters.json" \
   --parameters masterIgnition="${MASTER_IGNITION}" \ <1>
   --parameters sshKeyData="${SSH_KEY}" \ <2>
-  --parameters privateDNSZoneName="${CLUSTER_NAME}.${BASE_DOMAIN}" \ <3>
-  --parameters baseName="${INFRA_ID}"<4>
+  --parameters baseName="${INFRA_ID}"<3>
```

Comment:
```suggestion
  --parameters baseName="${INFRA_ID}" <3>
```

Reply:
One minor nit, otherwise lgtm!

Reply:
I recommend adding  a space before the callout for readability.

Reply:
@kmccarron-rh  Comment addressed.